content/en/docs/concepts/security/security-checklist.md

```diff
@@ -0,0 +1,378 @@
+- [ ] For nodes that support it, Seccomp is enabled with appropriate syscalls
+  profile for programs.
+- [ ] For nodes that support it, AppArmor or SELinux is enabled with appropriate
+  profile for programs.
+
+RBAC authorization is crucial but
+[cannot be granular enough to have authorization on the Pods' resources](/docs/concepts/security/rbac-good-practices/#workload-creation)
+(or on any resource that manages Pods). The only granularity is the API verbs
+on the resource itself, for example, `create` on Pods. Without
+additional admission, the authorization to create these resources allows direct
+unrestricted access to the schedulable nodes of a cluster.
+
+[Pod Security Admission](/docs/concepts/security/pod-security-admission/) is an
+admission controller to enable fine-grained authorization of Pod creation and
+updates. It replaces the Pod Security Policy admission controller.
+The [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
+define three different policies that restrict how fields can be set in the `PodSpec`.
+For a hands-on tutorial on Pod Security, see the blog post
+"[Kubernetes 1.23: Pod Security Graduates to Beta](/blog/2021/12/09/pod-security-admission-beta/)".
+
+{{< note >}}
+[PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/)
+will be removed in version 1.25.
+{{< /note >}}
+
+Pod Security admission `restricted` policy [can operate in several modes](/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces),
+`enforce`, `audit` or `warn` to help properly set the [security context](/docs/tasks/configure-pod-container/security-context/)
+according to security best practices. Nevertheless, pods' [security context](/docs/tasks/configure-pod-container/security-context/)
+should be separately investigated to limit the privileges and access pods may have.
+
+[Memory and CPU limits](/docs/concepts/configuration/manage-resources-containers/)
+should be set in order to restrict the memory and CPU resources a pod can
+consume on a node, and therefore prevent potential DoS attacks. It can be
+enforced by an admission controller.
+
+### Enabling Seccomp
+
+Seccomp can improve the security of your workloads by reducing the Linux kernel
+syscall attack surface available inside containers. The seccomp filter mode
+leverages BPF to create an allow or deny list of specific syscalls, named
+profiles. Those seccomp profiles can be enabled on individual workloads,
+[a security tutorial is available](/docs/tutorials/security/seccomp/). In
+addition, the [Kubernetes Security Profiles Operator](https://github.com/kubernetes-sigs/security-profiles-operator)
+is a project to facilitate the management and use of seccomp in clusters.
+
+Please note that Docker has been using
+[a default seccomp profile](https://docs.docker.com/engine/security/seccomp/)
+to only allow a restricted set of syscalls but Kubernetes is still not
+confining workloads by default. Fortunately,
```

Comment:
It might be worth replacing this with containerd specific content with the dockershim deprecation and removal being complete.

Reply:
Here's the default seccomp profile for containerd: https://github.com/containerd/containerd/blob/main/contrib/seccomp/seccomp_default.go

Reply:
Ok, I see! At first, I added that has historical context on the topic because I think Docker was one of the first to do that, and the idea was to say, "it was implemented way before Kubernetes and people should not assume it's present in Kubernetes".

Reply:
I tried to address that here https://github.com/kubernetes/website/pull/33992/commits/94bdce3c7c42526bd6649646cfc4a69c39d4eadd.